### PR TITLE
Wrap validation error instead of recreating

### PR DESCRIPTION
**Pull request by wangchenglong01**
## Which problem is this PR solving?
- Wrap validation error instead of recreating

## Short description of the changes
- Wrap validation error instead of recreating

Depends on submariner-io/releases#241

### CODE REVIEW

**Review by skitt — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Comment by wangchenglong01** [comment]
How to fix the failing of E2E?

**Comment by dfarrell07** [comment]
> How to fix the failing of E2E?

That vxlan+ovn test isn't marked as required because they are both new features and seem to still need work. Your PR very likely didn't impact that test, cause the failure. So no need to worry about it, we're just giving some time for folks to review. 

Although folks might be thinking about if this should go in now, right at the end of validating 0.11, or wait until after 0.11.  I'll leave that for others. :) /cc @tpantelis @maayanf24

**Comment by maayanf24** [comment]
> > How to fix the failing of E2E?
> 
> That vxlan+ovn test isn't marked as required because they are both new features and seem to still need work. Your PR very likely didn't impact that test, cause the failure. So no need to worry about it, we're just giving some time for folks to review.
> 
> Although folks might be thinking about if this should go in now, right at the end of validating 0.11, or wait until after 0.11. I'll leave that for others. :) /cc @tpantelis @maayanf24

Let's wait and merge this after the release of 0.11

File: pkg/routeagent_driver/cni/cni_iface.go [hunk 1]
```diff
@@ -91,13 +91,13 @@ func discover(clusterCIDR string) (*Interface, error) {
 func AnnotateNodeWithCNIInterfaceIP(nodeName string, clientSet kubernetes.Interface, clusterCidr []string) error {
 	cniIface, err := Discover(clusterCidr[0])
 	if err != nil {
-		return fmt.Errorf("DiscoverCNIInterface returned error %v", err)
+		return errors.Wrapf(err, "DiscoverCNIInterface returned error")
```
Comment (tpantelis) [inline pkg/routeagent_driver/cni/cni_iface.go]:
```suggestion
		return errors.Wrap(err, "Discover returned error")
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

**Review by tpantelis — APPROVED** [review]

**Pull request merged**